specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -70,6 +70,16 @@ Explain the user problem that need to be solved.
 
 ## High-level design
 
+#### Folder Structure
+
+* [frontend](/projects/frontend) The root folder for all frontend code.
+    * [shared-components](/projects/frontend/shared-components/README.md): the root folder for the shared components project, for more details please see the readme.
+      * gui: This contains the root package.json file for the project.
+        * projects:
+          * documentation-ui: functionality for testing and developing new components
```

Comment:
That's not completely correct now isn't . We plan to use not just for testing but as teh default Wrapper UI

Reply:
Not really. We plan on using the UI folder in the data-pipelines repo for the default wrapper ui.
Let me know if I can mark this as resolved?